src/osm_configurator/view/toplevelframes/create_project_frame.py

```diff
@@ -34,33 +42,72 @@ def __init__(self, state_manager: StateManager, project_controller: IProjectCont
             state_manager (state_manager.StateManager): The StateManager the frame will call, if it wants to change to another State.
             project_controller (project_controller.ProjectController): Respective controller
         """
-        window = super().__init__()  # Creates the window
+
+        # Creating the window
+        window = super().__init__(master=None,
```

Comment:
dont create a frame, it self shall be the frame, so go for self.super()__innit__